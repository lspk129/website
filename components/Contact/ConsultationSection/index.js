import styled from 'styled-components';

const ConsultationSection = styled.div`
  flex: 1;
  padding: 20px;
  background: #fff;
`;

export default ConsultationSection;
