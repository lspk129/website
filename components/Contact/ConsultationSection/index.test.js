import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ConsultationSection from './';

describe('ConsultationSection', () => {
  const shallowWrapper = shallow(<ConsultationSection />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<ConsultationSection />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('div');
  });
});
