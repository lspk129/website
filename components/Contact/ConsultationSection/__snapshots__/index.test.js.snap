// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsultationSection renders correctly 1`] = `
<div
  className="sc-bdVaJa cqoblB"
/>
`;
